catalog/templates/order.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Каталог</title>
+    <!--<link href="../css/bootstrap.min.css" rel="stylesheet"/>--><!--для локальной подгрузки стилей-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.7/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-LN+7fdVzj6u52u30Kp6M/trliBMCMKTyK833zpbD+pXdCLuTusPj697FH4R/5mcr" crossorigin="anonymous">
+</head>
+<body>
+    <div class="d-flex">
+        <!-- Боковое меню -->
+        <div class="bg-dark text-white p-3" style="width: 250px; height: 100vh;">
+            <h4 class="mb-4">Меню</h4>
+            <ul class="nav flex-column">
+                <li class="nav-item">
+                    <a class="nav-link active text-white" href="/home/">Главная</a>
```

Comment:
Главная страница на другом адресе
```suggestion
                    <a class="nav-link active text-white" href="{% url 'catalog:home' %}">Главная</a>
```